File: packages/canvas/src-native/canvas-android/canvas/src/main/java/com/github/triniwiz/canvas/HowToClear.java
package com.github.triniwiz.canvas;

enum HowToClear {
  // Skip clearing the backbuffer.
  Skipped,
  // Clear the backbuffer.
  JustClear,
  // Combine webgl.clear() API with the backbuffer clear, so webgl.clear()
  // doesn't have to call glClear() again.
  CombinedClear
}

File: packages/canvas/src-native/canvas-android/canvas/src/main/java/org/nativescript/canvas/HowToClear.kt
package org.nativescript.canvas

enum class HowToClear {
	// Skip clearing the backbuffer.
	Skipped,  // Clear the backbuffer.
	JustClear,  // Combine webgl.clear() API with the backbuffer clear, so webgl.clear()

	// doesn't have to call glClear() again.
	CombinedClear
}
